sample/BlobTrigger-CSharp/run.csx

```diff
@@ -2,10 +2,10 @@
 
 using Microsoft.WindowsAzure.Storage.Blob;
 
-public static void Run(CloudBlockBlob blob, CloudBlockBlob output, TraceWriter log)
+public static async Task Run(CloudBlockBlob blob, CloudBlockBlob output, TraceWriter log)
```

Comment:
The non async version still works, right? Why the change - just because async is better?

Reply:
Changed because the new storage APIs are async. Yes, non-async still works, but with the new APIs that would have made for a bad sample.